📝 Docs (#114)
cda-tum/qmap: docs/source/library/MappingResults.rst

Results
=======

This class captures additional results from the :code:`compile` method.

    .. currentmodule:: mqt.qmap
    .. autoclass:: MappingResults

In addition to the mapped circuit

    .. autoattribute:: MappingResults.mapped_circuit

> it also includes runtime information of the mapping procedure.

Reviewer: ```suggestion
it also includes runtime information of the mapping procedure
```

    .. autoattribute:: MappingResults.time
    .. autoattribute:: MappingResults.timeout

> Information about the input and output circuits are also stored.

Reviewer: ```suggestion
and information about the input and output circuits.
```

    .. autoattribute:: MappingResults.input
    .. autoattribute:: MappingResults.output

> If specified in the call to the :code:`compile` method, the `weighted max-SAT formula <http://www.maxhs.org/docs/wdimacs.html>`_ is also tracked.

Reviewer: ```suggestion
If specified in the call to the :code:`compile` method, the `weighted MaxSAT formula <http://www.maxhs.org/docs/wdimacs.html>`_ is also tracked.
```

Author: Done.
